opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -286,6 +290,23 @@ def detect(self) -> "Resource":
         return Resource(env_resource_map)
 
 
+class ProcessDetector(ResourceDetector):
```

Comment:
```suggestion
class ProcessResourceDetector(ResourceDetector):
```

Reply:
Done, it required a few more changes in various places though